caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -90,6 +92,9 @@ def env_var_to_int(name, default):
 # Batch size for encode() if <= 0 or invalid, the sentence-transformers default is used
 BATCH_SIZE = env_var_to_int("BATCH_SIZE", default=0)
 
+# To use dtype=bfloat16 for IPEX with autocast in encode.
+BFLOAT16 = os.getenv("BFLOAT16", "false").lower() not in FALSY
```

Comment:
There is already a `torch_dtype` config - https://github.com/caikit/caikit-nlp/blob/main/caikit_nlp/config/config.yml#L19 which users can use as env var `TORCH_DTYPE` so I'm worried that continuing to add these variables as env vars could potentially be confusing, especially when multiple module use is considered.

Perhaps a config section related to embeddings and/or ipex could be considered if there's expected to be these sorts of variables. That way for use as env vars they would automatically be pre-pended with `EMBEDDINGS_` or `IPEX_` etc.